internal/agent/executor/cache.go

```diff
@@ -194,6 +204,27 @@ func (executor *Executor) generateCacheKey(
 	return fmt.Sprintf("%s-%x", commandName, cacheKeyHash.Sum(nil)), true
 }
 
+func (executor *Executor) findLatestAvailableCache(ctx context.Context, uploader *LogUploader, commandName string) *api.CacheInfo {
+	cacheInfoRequest := api.CacheInfoRequest{
+		TaskIdentification: client.CirrusTaskIdentification,
+		CacheKey:           commandName + "-", // Use a prefix that will match any cache key for this command
```

Comment:
Should this use `CacheKeyPrefixes` instead of `CacheKey` if we want prefix matching to be used?